core/src/ibc/transaction_handler/mod.rs

```diff
@@ -59,7 +59,7 @@ pub fn execute(
             header,
```

Comment:
Make the UpdateClient function use the correnct method name in log
->
Make the UpdateClient function use the correct method name in log